applications/luci-app-cloudflared/root/usr/share/rpcd/acl.d/luci-app-cloudflared.json

```diff
@@ -0,0 +1,15 @@
+{
+	"luci-app-cloudflared": {
+		"description": "Grant access to Cloudflared configuration",
+		"read": {
+			"file": [ "/var/log/cloudflared.log" ],
```

Comment:
I think you'll also need a write file context to upload files to `/etc/cloudflared/`